dataikuapi/dss/savedmodel.py

```diff
@@ -315,9 +345,10 @@ def evaluate(self, dataset_ref):
 
         :meth:`set_core_metadata` must be called before you can evaluate a dataset
 
-        :param str dataset_ref: Name of the evaluation dataset to use (either a dataset name or "PROJECT.datasetName")
+        :param str dataset_ref: Evaluation dataset to use (either a dataset name, "PROJECT.datasetName", DSSDataset instance or dataiku.Dataset instance)
```

Comment:
ultra minor nitpick: make "DSSDataset" and "dataiku.Dataset" `:class:` so that they are hyperlinked in the doc

Reply:
ok